### ISSUE TITLE
Gemspec date is wrong

### ISSUE THREAD

**Issue by rbnpi** [Widdershin/sonic-pi-cli#17]
You haven't updated the gemspec date in the latest version
(if you read previous comment ignore.. had an aberration..doing too much python lately!)

**Comment by Widdershin** [comment]
Good point, haven't published a ruby package in a while so I forgot this was a thing. Will fix tonight.
